### PR TITLE
Updated keyserver url

### PR DESCRIPTION
**Pull request by bhasin85**
Please test it in your environment before the merge.

### CODE REVIEW

File: cookbooks/imos_postgresql/recipes/official_postgresql.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
   uri 'http://apt.postgresql.org/pub/repos/apt/'
   distribution "#{node['lsb']['codename']}-pgdg"
   components ['main']
-  keyserver 'keyserver.ubuntu.com'
+  keyserver 'hkp://keyserver.ubuntu.com:80'
```
Comment (pmbohm) [inline cookbooks/imos_postgresql/recipes/official_postgresql.rb]: If this is causing problems, would it also be causing the same problem in `..../chef/cookbooks/imos_java/recipes/default.rb`
?

**Comment by lwgordonimos** [comment]
Can this now be deleted? I'm not aware of any issues currently, and there was a change to update an expired key which may have been the cause of this original issue, https://github.com/aodn/chef/pull/315

**Comment by lwgordonimos** [comment]
@bhasin85 ?

**Comment by bhasin85** [comment]
Look like it got fixed with the other pull request you mentioned. I will close this one now.

**Pull request closed without merging**